bridges/snowbridge/pallets/inbound-queue-v2/Cargo.toml

```diff
@@ -53,6 +53,8 @@ hex-literal = { workspace = true, default-features = true }
 snowbridge-pallet-ethereum-client = { workspace = true, default-features = true }
 snowbridge-test-utils = { workspace = true }
 sp-keyring = { workspace = true, default-features = true }
+pallet-bridge-relayers = { workspace = true, default-features = true }
+pallet-bridge-messages = { workspace = true, default-features = true }
```

Comment:
Are these dependencies in use?

Reply:
Removed `pallet-bridge-messages` in https://github.com/vgeddes/polkadot-sdk/pull/23/commits/a60cfb1b9ceb48ed8a099c0e9537142232544b3d.
`pallet-bridge-relayers` is used here: https://github.com/claravanstaden/polkadot-sdk/blob/message-reward-topups/bridges/snowbridge/pallets/inbound-queue-v2/src/mock.rs#L36